test/integration/sponsors_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class SponsorsTest < ActionDispatch::IntegrationTest
+  test "should display same number of sponsors" do
+    visit root_path
+    click_link "Sponsors", match: :first
```

Comment:
It'll be good to use `I18n` here. https://github.com/ruby-bench/ruby-bench-web/blob/master/config/locales/en.yml#L74-L75